content/pt-br/docs/concepts/policy/node-resource-managers.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Gerenciamento de Recursos de Nós
```

Comment:
This should be "Gerenciadores", as you the original term is "Managers", instead of "Management".